Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated-NetFx_v4.x.nuspec

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.Azure.Functions.Worker.NetFx.ItemTemplates</id>
+    <version>1.0.$patchVersion$</version>
+    <authors>Microsoft</authors>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=2028464</licenseUrl>
+    <repository type="git" url="https://github.com/Azure/azure-functions-templates" />
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=848145</projectUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Collection of templates for Azure Functions .NET Isolated using .NET Framework</description>
+    <language>en-US</language>
+    <packageTypes>
+      <packageType name="Template" />
+    </packageTypes>
+  </metadata>
+  <files>
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-CSharp-Isolated\QueueTriggerCSharp.cs" target="content\QueueTrigger-CSharp\QueueTriggerCSharp.cs" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-CSharp-Isolated\.template.config\template.json" target="content\QueueTrigger-CSharp\.template.config\template.json" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-CSharp-Isolated\.template.config\vs-2017.3.host.json" target="content\QueueTrigger-CSharp\.template.config\vs-2017.3.host.json" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-CSharp-Isolated\.template.config\vs-2017.3\*.*" target="content\QueueTrigger-CSharp\.template.config\vs-2017.3" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-FSharp-Isolated\QueueTriggerFSharp.fs" target="content\QueueTrigger-FSharp\QueueTriggerFSharp.fs" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-FSharp-Isolated\.template.config\template.json" target="content\QueueTrigger-FSharp\.template.config\template.json" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-FSharp-Isolated\.template.config\vs-2017.3.host.json" target="content\QueueTrigger-FSharp\.template.config\vs-2017.3.host.json" />
+    <file src="..\..\..\Functions.Templates\Templates\QueueTrigger-FSharp-Isolated\.template.config\vs-2017.3\*.*" target="content\QueueTrigger-FSharp\.template.config\vs-2017.3" />
+    <file src="..\..\..\Functions.Templates\Templates\HttpTrigger-CSharp-Isolated-NetFx\HttpTriggerCSharp.cs" target="content\HttpTrigger-CSharp\HttpTriggerCSharp.cs" />
```

Comment:
All templates are the same except this one - points to `\HttpTrigger-CSharp-Isolated-NetFx\*` instead